### DISCUSSION TITLE
What is Recents?

### DISCUSSION THREAD

**Discussion by Spectre5** [immich-app/immich#14915] [Q&A]
On both iPhone and Android the first album you can select is "Recents".  What exactly is Recents?  I assumes this is more of a general phone question that Immich, but someone probably knows!  On my wife's iPhone the Recents has more assets than all other albums combined...

**Comment by alextran1502** [comment]
It includes everything

**Reply by Spectre5** [reply]
Do you know why it would include more than all of the others combined (by a lot)?

**Reply by alextran1502** [reply]
It is not the total of all media in all albums, it includes all files that are not in any album as well

**Reply by Spectre5** [reply]
Thanks.  And does it always include everything in all other albums?  In other words, if you select that then should you just skip all other selections?

**Reply by alextran1502** [reply]
Correct, we also have exclusion mechanism (double tap) so you can include everything but a certain albums

**Reply by Spectre5** [reply]
Got it thanks!  I didn't realize the difference between exclusion and not being selected until you said that.

**Reply by alextran1502** [reply]
No problem! Happy Holidays to you!

**Reply by Spectre5** [reply]
You too!  Glad to be a supporter/purchaser!  I hope your family is doing great this holiday season!

**Comment by Pe8er** [comment]
Sorry, this is still unclear to me...if I select just "Recents", I understand it will upload ALL of my photos. But will it create albums automatically?

And on the flip side: if I select "Recents" and a bunch of albums (that I had created), will photos be duplicated?
